languages/common-lisp/exercises/concept/sameness/.meta/design.md

```diff
@@ -0,0 +1,22 @@
+## Learning objectives
+
+- Be familiar with `eq`, `eql`, `equal` and `equalp` and how each builds upon the previous.
+- Understand that `=` works for numbers and does some 'conversion' of numbers
```

Comment:
We should probably remove this `=` objective now that it's part of the `arithmetic` exercise :)

Reply:
good point!